extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -448,6 +449,25 @@ describe('amp-a4a', () => {
       });
     });
 
+    it('for A4A layout should resolve once FIE is created', () => {
+      a4a.buildCallback();
+      a4a.onLayoutMeasure();
+
+      // Never resolve
+      sandbox.stub/*OK*/(FriendlyIframeEmbed.prototype,'whenIniLoaded')
+          .callsFake(() => {return new Promise(() => {});});
+      let creativeString = buildCreativeString();
+      // TODO: Remove after launch fie_ini_load_fix to 100%
+      creativeString = creativeString.replace('<body>',
```

Comment:
Couldn't find better way to mock the creative since they need to be signed. This is a workaround that will be removed soon.